pkg/gui/mergeconflicts/find_conflicts.go

```diff
@@ -47,14 +48,13 @@ func determineLineType(line string) LineType {
 	trimmedLine := strings.TrimPrefix(line, "++")
 
 	mapping := map[string]LineType{
-		"^<<<<<<< (HEAD|MERGE_HEAD|Updated upstream|ours)(:.+)?$": START,
-		"^=======$":    MIDDLE,
-		"^>>>>>>> .*$": END,
+		"<<<<<<< ": START,
+		"=======":  MIDDLE,
```

Comment:
On small thing here is that I have some files myself where I use equals as decoration e.g.:
```
=========================
HEADING
=========================
<body text>
```
If we're no longer asserting that the middle conflict marker has exactly 7 equals characters in a row, we might get some false positives. In that event, we'll actually cause a panic because we'll be doing `newConflict.middle = i` but newConflict won't be defined. So maybe we should go:
```go
if strings.HasPrefix(trimmedLine, "<<<<<<< ") {
  return START
} else if trimmedLine == "=======" {
  return START
} else if strings.HasPrefix(trimmedLine, ">>>>>>> ") {
  return END
}
```

i.e. the way you had it in the first commit haha